daemon/daemon.go

```diff
@@ -1040,15 +1041,25 @@ func (daemon *Daemon) Reload(config *Config) (err error) {
 	if config.IsValueSet("labels") {
 		daemon.configStore.Labels = config.Labels
 	}
+
 	if config.IsValueSet("debug") {
 		daemon.configStore.Debug = config.Debug
 	}
+
```

Comment:
(just quickly peaking at the code changes); these whitespace changes are not related to this change, and not everyone like adding too much whitespace; could you remove these?